packages/web3-auth/src/UI.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+
+const googleCardStyle = {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignItems: 'center',
+  width: '319px',
+  height: '45px',
+  background: '#252525',
+  borderRadius: '12px',
+  border: 'none',
+  cursor: 'pointer'
+} as React.CSSProperties
+
+const buttonTextSpan = {
+  fontStyle: 'normal',
+  fontWeight: 600,
+  fontSize: '18px',
+  lineHeight: '23px',
+  display: 'flex',
+  alignItems: 'center',
+  textAlign: 'center',
+  color: '#FFFFFF'
+} as React.CSSProperties
+
+const container = {
+  position: 'fixed',
+  float: 'left',
+  left: '50%',
+  top: '50%',
+  transform: 'translate(-50%, -50%)',
+  transition: 'opacity 400ms ease-in',
+  border: '1px solid #181818',
+  boxShadow: '5px 5px 0px #181818',
+  borderRadius: 10,
+  padding: 30,
+  zIndex: 100,
+  background: 'black'
+} as React.CSSProperties
+
+const footer = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  marginTop: '26.39px'
+} as React.CSSProperties
+
+const footerNormalText = {
+  fontStyle: 'normal',
+  fontWeight: '600',
+  fontSize: '14.109px',
+  color: '#535353',
+  lineGeight: '18px'
+} as React.CSSProperties
+
+const footerBigText = {
+  fontStyle: 'normal',
+  fontWeight: '700',
+  fontSize: '16.1351px',
+  lineHeight: '21px',
+  color: '#535353'
+} as React.CSSProperties
+
+const UI = (props: any) => {
+  return (
+    <div style={container}>
+      <img
+        src={'https://live-nft-hosted-assets.s3.ap-south-1.amazonaws.com/cancel_icon.svg'}
+        style={{ position: 'absolute', right: 18, cursor: 'pointer' }}
+        onClick={() => props.socialLogin.hideWallet()}
+      />
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          position: 'relative',
+          color: '#FFFFFF',
+          flexDirection: 'column'
+        }}
```

Comment:
Let's create a stylesheet and use classes.

Reply:
have created the stylesheets in https://github.com/bcnmy/biconomy-client-sdk/tree/webpack-changes
the webpack doesn't builds the css properly, let's merge this and look if we want to switch to stylesheet or not.